includes/handler.inc

```diff
@@ -454,11 +454,16 @@ function islandora_oai_query_solr($start = 0, $field = 'PID', $query = NULL, $se
     $query = '*:*';
   }
   else {
-    module_load_include('inc', 'islandora_solr', 'includes/utilities');
```

Comment:
I believe this is required for  islandora_solr_lesser_escape(). Should perhaps be at the top of the function rather than moved to the new conditional below.

Reply:
@willtp87 i feel that is what i requested, but my bad, did review on my phone screen while on a flight. opening a pull right now to fix. Thanks for catching this